### PR TITLE
[PKX-006] Pokemon Base Stats tab

### PR DESCRIPTION
**Pull request by giantiyan**

### CODE REVIEW

File: lib/utilities/extensions.dart [hunk 1]
```diff
@@ -137,4 +141,23 @@ extension StringExt on String {
...
+  String get shortcut => replaceAll('Special', 'Sp')
+      .replaceAll('attack', 'Atk')
+      .replaceAll('defense', 'Def');
+
+  Color get baseStatsColor {
+    var color;
+    var value = double.parse(this) / 200;
+
+    if (value < 0.25) {
+      color = Color(0xfffa6e70);
```
Comment (SaturnoMark) [inline lib/utilities/extensions.dart]: Please create a color.dart file 

Here is an example what color.dart file will contain
`const Color white = Color(0xFFFFFFFF);`

then you will only call that color into this line 
color = white

File: lib/features/base_stats_tab/base_stats_tab.dart [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+              value: baseStats?.base_stat?[3].toString(),
+            ),
+            BaseStatsInformation(
+              label: baseStats?.name?[4].toString().capitalize.shortcut,
+              value: baseStats?.base_stat?[4].toString(),
+            ),
+            BaseStatsInformation(
+              label: baseStats?.name?[5].toString().capitalize,
+              value: baseStats?.base_stat?[5].toString(),
+            ),
```
Comment (ryle-roasa) [inline lib/features/base_stats_tab/base_stats_tab.dart:41]: DRY, consider using map or mapIndexed

File: lib/features/base_stats_tab/widgets/base_stats_information.dart [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            flex: 2,
+            child: Text(
+              value ?? '',
+              style: TextStyle(fontSize: 16),
+            ),
+          ),
+          Expanded(
+            flex: 5,
+            child: ClipRRect(
+              borderRadius: BorderRadius.all(Radius.circular(10)),
```
Comment (ryle-roasa) [inline lib/features/base_stats_tab/widgets/base_stats_information.dart]: BorderRadius.circular(10)

File: lib/features/base_stats_tab/widgets/base_stats_information.dart [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+              child: LinearProgressIndicator(
+                value: double?.parse(value ?? '0') / 200,
+                valueColor: new AlwaysStoppedAnimation<Color>(
+                    (value ?? '0').toString().baseStatsColor),
```
Comment (ryle-roasa) [inline lib/features/base_stats_tab/widgets/base_stats_information.dart]: Could you use the Pokemon type color here

**Review by ryle-roasa — APPROVED** [review]

**Review by SaturnoMark — APPROVED** [review]

**Pull request merged**